sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_AnalyzeOperation.md

```diff
@@ -0,0 +1,111 @@
+# Running Analyze Operation Asynchronously
+This sample demonstrates how to run analyze operation in one or more documents and get them asynchronously for multiple tasks including PII, NER and KPE . To get started you will need a Text Analytics endpoint and credentials.  See [README][README] for links and instructions.
+
+## Creating a `TextAnalyticsClient`
+
+To create a new `TextAnalyticsClient` to run analyze operation for a document, you need a Text Analytics endpoint and credentials.  You can use the [DefaultAzureCredential][DefaultAzureCredential] to try a number of common authentication methods optimized for both running as a service and development.  In the sample below, however, you'll use a Text Analytics API key credential by creating an `AzureKeyCredential` object, that if needed, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:TextAnalyticsSample4CreateClient
+var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+```
+
+## Running Analyze Operation Asynchronously in multiple documents
+
+To run analyze operation in multiple documents, call `StartAnalyzeOperationBatchAsync` on an `IEnumerable` of strings.  The result is a Long Running operation of type `AnalyzeOperation` which polls for the results from the API.
+
+```C# Snippet:AnalyzeOperationBatchAsync
+    string document = @"We went to Contoso Steakhouse located at midtown NYC last week for a dinner party, 
+                        and we adore the spot! They provide marvelous food and they have a great menu. The
+                        chief cook happens to be the owner (I think his name is John Doe) and he is super 
+                        nice, coming out of the kitchen and greeted us all. We enjoyed very much dining in 
+                        the place! The Sirloin steak I ordered was tender and juicy, and the place was impeccably
+                        clean. You can even pre-order from their online menu at www.contososteakhouse.com, 
+                        call 312-555-0176 or send email to order@contososteakhouse.com! The only complaint 
+                        I have is the food didn't come fast enough. Overall I highly recommend it!";
+
+    var batchDocuments = new List<TextDocumentInput>
```

Comment:
the description talks about ``IEnumerable` of strings` but this is a ``IEnumerable` of TextDocumentInput`.
It will be better to use the string one as it is more user friendly

Reply:
updated